### ISSUE TITLE
Installed apps not showing as installed

### ISSUE THREAD

**Issue by vertigo220** [rzander/ruckzuck#125]
I went through all the apps in RZ and the following are installed on my computer but not showing as such in RZ in the install list (some of them (*) are showing as having updates in that list (others are probably just up-to-date), so in one place RZ sees them and in the other it doesn't):

Firefox
Vivaldi
Krita*
Signal*
Slack
GOG Galaxy* (possibly because I'm running v1.x instead of 2.x, but that shouldn't matter; also, this should be under Games section, as should Steam)
LockHunter*
OpenVPN* (should be in VPN section)
.NET 4.8 & 3.5
Ghostscript*
Gpg4win*
KeePass
CDBurnerXP
Etcher
f.lux*
WinDirStat*
WinSCP*
PotPlayer* (Potplayer)
Subtitle Edit
Picard*
eMClient*
MediaMonkey*
LAV Filters*
Thunderbird*
AIMP*
foobar2000*
Edge*
Mp3tag*

**Comment by rzander** [comment]
1. Are there any Package where you expect an update but RZ will not offer one ? then I need ProductName, ProductVersion and Manufacturer as it's listed in your AddRemove Programs...
2. The ones with a * are up to date, but do not show as installed (green) in the RZ UI ? correct ?

Gog, Steam and OpenVPN have been moved to the correct section.

**Comment by vertigo220** [comment]
1. Haven't checked that, but I can.
2. No, they're apps that RZ is showing have updates available, so it clearly sees they're installed, yet they aren't shown as installed on the install screen (not highlighted green).

**Comment by vertigo220** [comment]
The only apps RZ sees as installed but doesn't show updates for are all browsers: Brave, Vivaldi, and Waterfox.

By the way, assuming it's helpful for users to check that occasionally (and I'm guessing it is), there are a couple things that make it much more difficult to do than it could/should be:

- the apps listed in the update screen aren't in alphabetical order, so comparing the list against SUMo was a bit difficult as I had to constantly scroll up and down in RZ looking for the apps.
- I had to constantly switch back and forth between the install screen and update screen to check if apps that were showing as out-of-date in SUMo but not in RZ were being missed by RZ or are not even handled by it.

Suggested fixes:
- alphabetize the update list
- add a search bar at the top of the update list which can be used to search for an app in RZ's repo (to see if it's one handled by RZ) --OR (and probably better)--
- color apps in the install list yellow if they're installed and need an update, so the user can just search for each app shown needing an update in SUMo and see if they're green, yellow, or not colored.

Going through the list also reminded me that PotPlayer is going to always show as needing an update, since I'm staying on an older version. The reason for this is that after the 2019-06 (1.7.18958) version, ads were placed in it, but that's not the worst of it: they're in an invisible window that hides in the corner of the screen, blocking stuff behind it (from clicking, things can still be seen since the window is invisible) and showing hidden web content which is a security risk and can potentially be clicked without realizing it. So it's a somewhat similar situation to the WinMerge discussion, and I wonder what the appropriate action is as far as how to deal with the package in RZ, whether to post a warning, set it to use the last non-adware version, or do nothing.
PP
